app/antedecorators/authz_nested_message.go

```diff
@@ -8,6 +8,9 @@ import (
 	evmtypes "github.com/sei-protocol/sei-chain/x/evm/types"
 )
 
+// maxNestedMsgs defines a cap for the number of nested messages on a MsgExec message
+const maxNestedMsgs = 7
```

Comment:
lets reduce this since even `7` seems excessive for a normal use case, maybe 5?

Reply:
sure

Reply:
it looks the golangci-lint / lint failing due to new golangci-lint version update yesterday. I will cut a ticket to fix it. @udpatil